torch_geometric/datasets/mesh_cnn_datasets.py

```diff
@@ -0,0 +1,210 @@
+from torch_geometric.datasets.mesh_cnn_base_datasets import (
+    MeshCnnClassificationDataset, MeshCnnSegmentationDataset)
+
+
+class MeshShrech16Dataset:
```

Comment:
We can probably clean this up to avoid a lot of duplicated code. What about only providing `MeshCNNClassificationDataset` and `MeshCNNSegmentationDataset` in which you can decide on which dataset to use, *e.g.*:
```python
dataset = MeshCNNClassificationdataset(root, name='shrech16',
                                       train=True, transform=MeshCNNPrepare(...))
```

Reply:
Yes you are right, but I meant to create base classes such as MeshCNNClassificationdataset and a specific classes for specific datasets (such as shrec16, cubes...). Do you think I should remove it? Please see the fix I did in my last commits.